scripts/build_android_libtorch_and_pytorch_android.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+set -eux
```

Comment:
nit: the name of the script is a bit verbose - can we simple call it "build_pytorch_android.sh"?
you can put a comment here to describe what it does.

Reply:
done